### PR TITLE
Event improvement

### PR DESCRIPTION
**Pull request by SaschaMzH**
Now it is possible to set the position and the size to an event object.

Braking changes:

- Button object renamed to ButtonEvent
- Each event object need a position and a size

### CODE REVIEW

**Comment by SaschaMzH** [comment]
[Position for event objects](https://trello.com/c/kAbcSYqt/26-position-for-event-objects)

File: python_lib/EventSystem.py [hunk 1]
```diff
@@ -14,17 +14,29 @@
...
+    """ Base class for all event based objects.
     """
     callback = None
     x = 0
     y = 0
+    width = 1
+    height = 1
 
-    def __init__(self, register_callback, x=1, y=1):
+    def __init__(self, register_callback, x, y, width, height):
```
[the remark this answers is not in the document]
Comment (SaschaMzH) [reply python_lib/EventSystem.py]: Renamed to ControlBase

File: python_lib/EventSystem.py [hunk 2]
```diff
@@ -14,17 +14,29 @@
...
         self.callback = register_callback
         self.x = x
         self.y = y
+        self.width = width
+        self.height = height
+
+
+class ButtonEvent(ControlBase):
```
Comment (JonasTrampe) [inline python_lib/EventSystem.py]: Isn't this just a button?

Comment (JonasTrampe) [reply python_lib/EventSystem.py]: I am marking this as a todo.

**Pull request merged**